docs/IcepeakProtocol.md

```diff
@@ -19,3 +29,42 @@ Example:
 3. The server immediately sends the current value of `/path/to/channel`, e.g. `{"status": "updating"}`
 4. The value gets updated to `{"status": "done"}` (by another process)
 5. The server then immediately sends the updated value to all clients listening to `/path/to/channel`
+
+### Modifying data
+
+Data on Icepeak can be modified by either a PUT request or a DELETE request. The PUT will introduce
+or overwrite new information to the Icepeak server and DELETE will remove information from the server.
+In case some information changes, all information that is changed will be send to every 
+client that is listening to that channel. 
+
+- `Put` writes a JSON value to a given path in the JSON structure.
+
+  - If the path descends into a JSON value that is not an object, that value is
+    replaced by an object that just contains the newly created path.
+  - If a value along the path is already an object, the field specified by the
+    path is added to it.
+
+- `Delete` removes a JSON value at a given path.
+
+  - If the path does not exist, nothing happens.
+  - If the path refers to a field in an object, that field is removed from the
+    object.
+
+Both of these operations have an optional parameter, called `durable`, which if set,
```

Comment:
What does being "set" mean in this context. It could be
- `foo/bar/baz?durable`
- `foo/bar/baz?durable=true`
- `foo/bar/baz?durable=yes`
- `foo/bar/baz?durable=1`
- etc.

Please specify which of these is meant.